src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -244,6 +245,46 @@ async Task<HttpResponseMessage> SendRequest(HttpRequestMessage request)
             }
         }
 
+        // timeout is 300s and no retry on timeout
+        private async Task<string> LongFetch(Func<HttpRequestMessage> requestFactory, string? value404 = null, HttpMiddleware? middleware = null)
```

Comment:
instead of copying this method, dd an optional `HttpClient httpClient = null` parameter to the `Fetch` method and use it over `_http` if present.

Reply:
We could disable retry entirely for `OnDemandHierarchyDrySync`

Reply:
ok